index.html

```diff
@@ -14,8 +14,16 @@
         </header>
         <main>
             <section class="main-content">
-                <div class="worker-container"></div>
-                <div class="client-container"></div>
+                <h1 class="main-logo">
+                    <p class="logo-sub"><img src="./img/main-logo-sub.svg" alt="サイトタイトル"></p>
+                    <p class="title-logo"><img src="./img/main-logo.svg" alt="サイトタイトル"></h1>
```

Comment:
MUST
`<p>`が閉じていないです。

マークアップのバリデーションツールを使うと良いかもです。
[ref](https://validator.w3.org/#validate_by_input)
ref先のinputにマークアップをコピペすると、バリデーションしてくれます＾＾

```suggestion
                    <p class="title-logo"><img src="./img/main-logo.svg" alt="サイトタイトル"></p>
```

Reply:
コメントありがとうございます！
修正しました。
バリデーションツールも活用させていただきます！

3a13af9